### ISSUE TITLE
Oracle - UpdateToTagSQL - Issue with delimeters

### ISSUE THREAD

**Issue by metaloid2k21** [liquibase/liquibase#1879]
## Environment
**Liquibase Version**: 4.3.5

**Liquibase Integration & Version**:  CLI

**Liquibase Extension(s) & Version**: 

**Database Vendor & Version**: Oracle 19.5.0.0

**Operating System Type & Version**: Windows 10 Pro

## Description
I’ve “base” DEV and QA envs which are fully CI integrated using liquibase. When I try to use the UpdateTagSQL command to provide a “custom” env patch overview + installation script, I’ve the following issue with the generated code:

every plsql block that needs a newline+/ (slash) char in the end to work, gets generated with no newline char. example:

*  changeset instruction

`<sqlFile … endDelimiter="/"…
`

* generated PLSQL instruction

`END;/
`

This fails when executing trough and Oracle compatible SQL Client (event sqlplus).

## Steps To Reproduce
* Use UpdateTagSQL command over a changelog containing sqlFile tag. The included SQL script should be terminated with "/" char (example - PLSQL procedure create instruction):

```
<ns0:databaseChangeLog xmlns:ns0="http://www.liquibase.org/xml/ns/dbchangelog"
    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext  http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd  http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
    <ns0:changeSet author="jmalbuquerque" id="REQ123_28052021_45" runOnChange="true">
        <ns0:sqlFile encoding="utf8" endDelimiter="/" path="../../SCH_ADMIN/SCRIPTS/REQ123/REQ123_45_PROCEDURE_lock_table_online_create_INSTALL.sql" relativeToChangelogFile="true" stripComments="false" />
        <ns0:rollback>
            <ns0:sqlFile encoding="utf8" endDelimiter="/" path="../../SCH_ADMIN/SCRIPTS/REQ123/REQ123_45_PROCEDURE_lock_table_online_create_UNINSTALL.sql" relativeToChangelogFile="true" stripComments="false" />
        </ns0:rollback>
    </ns0:changeSet>
  <ns0:changeSet author="pjalmeida" id="TAG213">
        <ns0:tagDatabase tag="TAG213" />
    </ns0:changeSet>
</ns0:databaseChangeLog>
```

## Actual Behavior
Generated SQL script fails when executing over target database (syntax errors)

## Expected/Desired Behavior
Being able to use the UpdateTAGSQL feature to generate valid and executable SQL code.

## Screenshots (if appropriate)
NA

## Additional Context
Already submitted this question in community forum: https://forum.liquibase.org/t/oracle-updatetagsql-issue-with-delimeters/5538/4

**Comment by molivasdat** [comment]
Example test.sql file
```
DECLARE
e_error EXCEPTION;
PRAGMA exception_init ( e_error,-00942 );
BEGIN
EXECUTE IMMEDIATE ‘drop table AD_GROUP_ROLE_MAPPING’;

EXCEPTION
WHEN e_error THEN
NULL;
END;

/
CREATE TABLE AD_GROUP_ROLE_MAPPING
(
AD_GROUP_NAME VARCHAR2(200) NOT NULL,
ROLE_ID NUMBER,
PRIMARY KEY (AD_GROUP_NAME)
)
```

**Comment by FBurguer** [comment]
UpdateToTagSQL generates a correct sql except that it includes the `/` when it was declared as an enddelimiter. Maybe we can look at it as an enhancement?

**Comment by FBurguer** [comment]
Looks like its solved for liquibase `4.15`. Im closing the issue but please let us now if this is still an issue for you and we will have a second look, thanks!
